assets/data/events.json

```diff
@@ -4843,7 +4843,7 @@
     "when": "05:45 PM-07:15 PM",
     "eventType": "Class/Workshop",
     "what": "qualiTEA touch — A consent driven, hands on—or hands off—embodiment workshop, facilitated by licensed massage therapists. We invite participants to mindfully engage with self, partnered, or group touch. Through deep listening/experiencing, we connect conscious understanding with nervous system knowing. Come for attuned bodywork, stay for discussion and tea service to follow. Seating",
-    "where": "Fortunia Tea House",
```

Comment:
Is this required for this change?

Reply:
this is a sneaky change for my theme camp! all our other events are under the camp name "Fortunia".

Reply:
@Boneill3 fixed the branch base, thanks for pointing it out 🙂